governance/contract.go

```diff
@@ -121,7 +121,7 @@ func (e *ContractEngine) contractAddrAt(num uint64) (common.Address, error) {
 	// this happens when GovParamContract has not been voted
 	param, ok := headerParams.Get(params.GovParamContract)
 	if !ok {
-		logger.Error("Could not find GovParam contract address")
+		logger.Debug("Could not find GovParam contract address")
```

Comment:
how about logger.Info? I believe that this kind of information is shown at the Info level.

Reply:
The log is printed when (1) the Kore block has passed AND (2) "governance.govparamcontract" is empty, which is a very ordinary situation. This log is printed about 5 times per block. It will quickly clog the log file.

In contrast, an Info log is printed if the AddressBook is not ready (https://github.com/klaytn/klaytn/blob/dev/reward/addressbook_connector.go#L205).
It is okay to use Info-level here because the log is only printed every 86400 blocks.

Reply:
I see. My concern is when I run some nodes privately for testing purposes, I imagine just forgetting to add the contract address and I do not have any clue whether I did deploy it or not.

Reply:
In that case, you can verify the output of `governance.itemsAt` along with other test preconditions.

Reply:
If the "governance.govparamcontract" points to a broken contract address, other Warn or Error logs will be printed in the contractCaller.

Reply:
Thank you for your detailed explanation!